bundles/org.openhab.binding.folderwatcher/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="folderwatcher" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>FolderWatcher Binding</name>
+	<description>This is the FolderWatcher binding.</description>
+	<author>Alexandr Salamatov</author>
```

Comment:
The author tag is no longer supported. Please remove it.

Reply:
removed